test/Renci.SshNet.Tests/Classes/SessionTest_Connected_ConnectionReset.cs

```diff
@@ -64,8 +64,6 @@ public void ErrorOccurredIsRaisedOnce()
 
             var connectionException = (SshConnectionException)exception;
             Assert.AreEqual(DisconnectReason.ConnectionLost, connectionException.DisconnectReason);
-            Assert.IsNull(connectionException.InnerException);
-            Assert.AreEqual("An established connection was aborted by the server.", connectionException.Message);
```

Comment:
not always the case - it could contain a SocketException upon RST, in which case the message is more like "An existing connection was forcibly closed by the remote host."